server/channels/app/plugin_api_tests/manual.test_http_hijack_plugin/main.go

```diff
@@ -26,8 +27,16 @@ func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, _ *http.Req
 		return
 	}
 
-	conn.Write([]byte("HTTP/1.1 200\n\nOK"))
-	conn.Close()
+	_, err = conn.Write([]byte("HTTP/1.1 200\n\nOK"))
+	if err != nil {
+		mlog.Error("Failed to write to connection", mlog.Err(err))
```

Comment:
Let's return an `w.WriteHeader(http.StatusInternalServerError)`. Otherwise the test won't fail, if this errors.

Reply:
committed the change